localstack/services/sqs/provider.py

```diff
@@ -415,6 +443,9 @@ def get(self, block=True, timeout=None, visibility_timeout: int = None) -> SqsMe
 
             return copied_message
 
+    def generate_receipt_handle(self, message: SqsMessage) -> str:
+        return generate_receipt_handle(self.arn, message)
```

Comment:
That took me a second look to realize that this is not an endless recursion^^ Maybe because it's Friday, or maybe because they look similar on first glance, so might be a candidate for function renaming.